ipatests/test_integration/test_otp.py

```diff
@@ -83,13 +85,72 @@ def kinit_otp(host, user, *, password, otp, success=True):
     )
 
 
+def ssh_2fa_with_cmd(host, username, password, otpvalue, command="exit 0"):
+    """ ssh to user and in same session pass the command to check tgt of user
+    :param host: host to ssh
+    :param str username: The name of user
+    :param str password: password, usually the first factor
+    :param str otpvalue: generated pin of user
+    :param str command: command to execute in same session,
+     by deafult set to "exit 0"
+    :return: object class of expect command run
+    """
+    tasks.install_packages(host, ['expect'])
```

Comment:
If you need to install an additional paclage, it's better to do it before the server installation (in the `install` method of `class TestOTPToken`, just before the call to `super(TestOTPToken, cls).install(mh)`. This ensures that the external DNS is used.

Reply:
Please check, updated.